modular_skyrat/modules/cargo/code/bloat.dm

```diff
@@ -46,6 +46,13 @@
 					/obj/item/clothing/suit/armor/vest/alt)
 	crate_name = "armor crate"
 
+/datum/supply_pack/security/helmets
+	name = "Helmets Crate"
+	desc = "Contains three surplus brain buckets. Requires Security access to open."
+	cost = CARGO_CRATE_VALUE * 2
+	contains = list(/obj/item/clothing/head/helmet,
+					/obj/item/clothing/head/helmet,
+					/obj/item/clothing/head/helmet)
```

Comment:
```suggestion
					/obj/item/clothing/head/helmet,)
```
trailing comma good :)

Reply:
when everything else in bloat does it, maybe, but consistency is king